mlir/tools/mlir-clang/Test/polybench/run-all-clang.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+export PATH=$HOME/MLIR-GPU/build/bin:$PATH
+
+CC='clang -O3'
+LDFLAGS=-lm
+BASE=$(pwd) 
+
+dirList="linear-algebra/blas 
+         linear-algebra/kernels 
+         linear-algebra/solvers 
+         datamining
+         stencils
+         medley"
+
+for dir in $dirList; do
+  if [ -d $dir ]; then
+    cd $dir
+    #echo $dir
+    for subDir in `ls`; do
+      cd $subDir
+      #echo "compiling in dir -> $(pwd)"
+      S=$CC
+      S+=" -I $BASE/utilities -I $(pwd)"
+      S+=" $BASE/utilities/polybench.c $subDir.c -DPOLYBENCH_TIME -DPOLYBENCH_NO_FLUSH_CACHE $LDFLAGS -o $subDir.exe"
```

Comment:
If helpful I believe clang supports the -D WITH_SPACE that mlir-clang requires.

Reply:
Perfect thanks. I will merge the scripts later on. BTW do we want to take some results? I think everything is ready